README.rst

```diff
@@ -1,110 +1,92 @@
-urllib3
-=======
+hip
+===
 
-.. image:: https://travis-ci.org/urllib3/urllib3.svg?branch=master
+.. image:: https://travis-ci.org/python-trio/hip.svg?branch=master
         :alt: Build status on Travis
-        :target: https://travis-ci.org/urllib3/urllib3
+        :target: https://travis-ci.org/python-trio/hip
 
-.. image:: https://img.shields.io/appveyor/ci/urllib3/urllib3/master.svg
-        :alt: Build status on AppVeyor
-        :target: https://ci.appveyor.com/project/urllib3/urllib3
+.. image:: https://github.com/python-trio/hip/workflows/CI/badge.svg
+        :alt: Build status on GitHub Actions
+        :target: https://github.com/python-trio/hip/actions
 
-.. image:: https://readthedocs.org/projects/urllib3/badge/?version=latest
-        :alt: Documentation Status
-        :target: https://urllib3.readthedocs.io/en/latest/
-
-.. image:: https://img.shields.io/codecov/c/github/urllib3/urllib3.svg
+.. image:: https://img.shields.io/codecov/c/github/python-trio/hip.svg
         :alt: Coverage Status
-        :target: https://codecov.io/gh/urllib3/urllib3
+        :target: https://codecov.io/gh/python-trio/hip
 
-.. image:: https://img.shields.io/pypi/v/urllib3.svg?maxAge=86400
+.. image:: https://img.shields.io/pypi/v/hip.svg?maxAge=86400
         :alt: PyPI version
-        :target: https://pypi.org/project/urllib3/
+        :target: https://pypi.org/project/hip/
 
-.. image:: https://badges.gitter.im/python-urllib3/Lobby.svg
+.. image:: https://badges.gitter.im/python-trio/hip.svg
         :alt: Gitter
-        :target: https://gitter.im/python-urllib3/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+        :target: https://gitter.im/python-trio/hip
 
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
     :target: https://github.com/psf/black
 
-urllib3 is a powerful, *sanity-friendly* HTTP client for Python. Much of the
-Python ecosystem already uses urllib3 and you should too.
-urllib3 brings many critical features that are missing from the Python
-standard libraries:
+hip is a new Python HTTP client for everybody. It supports synchronous Python (just like requests does), but also Trio, asyncio and Curio.
```

Comment:
Let's use "Hip" instead of "hip" for the name of the library in copy?

Reply:
Oops, sorry for Hip, fixed. I tried adding a disclaimer but I don't know how to make it look good in a .rst file. Maybe it should just be simple text.